crates/swc_css_parser/tests/recovery/at-rule/extra-semi/output.json

```diff
@@ -2,7 +2,7 @@
   "type": "Stylesheet",
```

Comment:
Changes expected, because now we output list of component values when we can't parser (as in spec)